players.yaml

```diff
@@ -0,0 +1,8 @@
+activePlayers:
+  - name:  ArchmageInc
+    points: 10
+  - name: jimmyhmiller
+    points: 0
+  - name: dilbert88 
+    points: 0
+inActivePlayers: []
```

Comment:
Yes it should be inactivePlayers. Will fix.
On Mar 8, 2016 2:01 PM, "Robert Long" <notifications@github.com> wrote:

> In players.yaml
> <https://github.com/nomicness/a-whole-new-world/pull/8#discussion_r55408858>
> :
>
> > @@ -0,0 +1,8 @@
> > +activePlayers:
> > +  - name:  ArchmageInc
> > +    points: 10
> > +  - name: jimmyhmiller
> > +    points: 0
> > +  - name: dilbert88
> > +    points: 0
> > +inActivePlayers: []
>
> FYI, camelCase makes inActivePlayers a bit confusing to read. I'm not
> saying you should change it. I'm just making an observation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nomicness/a-whole-new-world/pull/8/files#r55408858>.
>